sensors/new_plexxi_sensor.py

```diff
@@ -0,0 +1,128 @@
+# -*- coding: utf-8 -*-
```

Comment:
Why is this file called new_plexxi_sensor.py?  plexxi no longer exists. Please use the proper product name.